core/src/main/scala/cromwell/core/package.scala

```diff
@@ -0,0 +1,17 @@
+package cromwell
+
+import wdl4s.values.{SymbolHash, WdlValue}
+
+import scalaz._
+
+/**
+  * Created by tjeandet on 3/21/16.
+  */
+package object core {
+  type ErrorOr[+A] = ValidationNel[String, A]
+
+  type LocallyQualifiedName = String
+  case class CallOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
+  type CallOutputs = Map[LocallyQualifiedName, CallOutput]
+
```

Comment:
take this \n and stick it after the }, and then you'll have made both myself and khalid happy